casper/order/order_view.py

```diff
@@ -0,0 +1,73 @@
+"""The blockchain view module extends a view for blockchain data structures """
+from casper.safety_oracles.clique_oracle import CliqueOracle
+from casper.abstract_view import AbstractView
+from casper.order.bet import Bet
+import casper.order.order_estimator as estimator
+
+import random as r
+
+
+class OrderView(AbstractView):
+    LIST = ["dog", "frog", "horse", "pig", "rat", "whale", "cat"]
```

Comment:
We can wait to add this as something generated by the inital_message function, and for subsequent validators give it a random order :)

Reply:
Also, if this is the only change in the view between this and Binary (or Integer), we can merge after merging :)

Reply:
agreed